src/sources/docker_logs.rs

```diff
@@ -79,6 +80,7 @@ pub struct DockerTlsConfig {
 impl Default for DockerLogsConfig {
     fn default() -> Self {
         Self {
+            host_key: None,
```

Comment:
If `host_key` has a default value (by the docs) why rust `Default` return `host_key` as `None`? I think it's better remove `Option` and add `#[serde(default = default_value_fn)]`.

Reply:
Thanks!